generators/chipyard/src/main/scala/config/RoCCAcceleratorConfigs.scala

```diff
@@ -56,3 +56,9 @@ class HwachaLargeBoomConfig extends Config(
   new boom.common.WithNLargeBooms(1) ++
   new chipyard.config.WithSystemBusWidth(128) ++
   new chipyard.config.AbstractConfig)
+
+class AES256ECBRocketConfig extends Config(
+  new aes.WithAES256ECBAccel ++                                   // use Caliptra AES 256 ECB accelerator
+  new freechips.rocketchip.subsystem.WithNBigCores(1) ++
+  new chipyard.config.WithSystemBusWidth(128) ++
```

Comment:
Does this require 128b sbus to function correctly? We should try to isolate system-bus configuration from core/accelerator configurations.

Reply:
I think it requires a 256b `sbus` right now since the `L2MemHelper` code is specifically made for that. But yes I agree with you.

Reply:
I abstracted out the `rocc-acc-utils` code to be SBUS agnostic but kept a require for this accelerator to use a 256b SBUS.

Reply:
Is that a performance requirement, or a functionality requirement?

Reply:
Functionality requirement right now. It could be abstracted with a bit more work to be a performance requirement. That's for a later day